src/webui/api/torrentscontroller.cpp

```diff
@@ -2143,3 +2143,20 @@ void TorrentsController::onMetadataDownloaded(const BitTorrent::TorrentInfo &inf
             iter.value().setTorrentInfo(info);
     }
 }
+
+void TorrentsController::setCommentAction()
+{
+    requireParams({u"hash"_s, u"comment"_s});
+
+    const auto id = BitTorrent::TorrentID::fromString(params()[u"hash"_s]);
+
+    BitTorrent::Torrent *const torrent = BitTorrent::Session::instance()->getTorrent(id);
+    if (!torrent)
+        throw APIError(APIErrorType::NotFound);
+
+    QString comment = params()[u"comment"_s].trimmed();
+    comment.replace(QRegularExpression(u"\r?\n"_s), u" "_s);
```

Comment:
I would still suggest you follow the coding style as suggested in https://github.com/qbittorrent/qBittorrent/pull/23031#discussion_r2231821950